src/cmds/scripts/win_postinstall.py

```diff
@@ -54,6 +54,7 @@
 pbs_sbin = None
 installtype = 'execution'
 server = None
+scp_path = 'CHANGE_THIS_TO_PATH_TO_SCP_COMMAND'
```

Comment:
@sakshamgarg what will happen if the admin does not specify or change the scp_path. Can you please post some results on how well does PBS handle an invalid scp_path

Reply:
When the admin does not specify scp_path, a message is printed on the std output stating this:

```
NOTICE :
Path to SCP command is not specified.
Modify the path manually in C:\Program Files (x86)\PBS\pbs.conf.
Restart pbs services after updating.
```

This is also shown in the logs attached above. If the admin wants to change the path after running postinstall script, then they have to manually change the path in pbs.conf file.

The postinstall script doesn't check the validity of the scp_path specified by the admin. But if the path is wrong, then stageout/stagein of files will fail. For example, the job output and error files will not be copied to client and the tracejob command will give "unable to copy file" error. This is how PBS handles an invalid scp_path. Below is the output of the tracejob of a job when an invalid scp_path was given.

```
C:\Program Files (x86)\PBS\exec>tracejob 2011

Job: 2011.saksham

10/24/2019 01:13:58  M    allowed saksham to access window station and desktop, User saksham passworded
10/24/2019 01:13:58  M    Started, pid = 5212
10/24/2019 01:13:58  M    Started, pid = 5212
10/24/2019 01:13:58  M    task 1 terminated
10/24/2019 01:13:58  M    Terminated
10/24/2019 01:13:58  M    task 00000001 cput= 0:00:00
10/24/2019 01:13:58  M    kill_job
10/24/2019 01:13:58  M    winbuild cput= 0:00:00 mem=0kb
10/24/2019 01:13:58  M    Obit sent
10/24/2019 01:13:58  M    copy file cred request received
10/24/2019 01:13:58  M    User saksham passworded
10/24/2019 01:14:35  M    Unable to copy file C:/PROGRA~2/PBS/home/spool/2011.saksham.OU to wind10:C:/Users/saksham/STDIN.o2011
10/24/2019 01:14:35  M    wind10: Unknown error
10/24/2019 01:15:12  M    Unable to copy file C:/PROGRA~2/PBS/home/spool/2011.saksham.ER to wind10:C:/Users/saksham/STDIN.e2011
10/24/2019 01:15:12  M    wind10: Unknown error
10/24/2019 01:15:12  M    staged 2 items out over 0:01:14
10/24/2019 01:15:22  M    post_cpyfile: entered 2011.saksham
10/24/2019 01:15:22  M    post_cpyfile: done 2011.saksham
10/24/2019 01:15:23  M    delete job request received
10/24/2019 01:15:23  M    kill_job
`````